test/cli/test_cli_functions.sh

```diff
@@ -322,3 +321,61 @@ function check_deletes {
     delete_backingstore_path
     echo_time "✅  delete cycle is done"
 }
+
+function crd_arr { 
+    crd_array=($(kubectl get crd | awk '{print $1}' | grep -v "NAME"))
+    echo_time "${crd_array[*]}"
+}
+
+function noobaa_uninstall {
+    check_cleanflag=$((RANDOM%2))
+    echo_time ${check_cleanflag}
+    
+    if [ ${check_cleanflag} -gt 0 ] 
+    then
+        echo_time "Running uninstall with --cleanup”"
+        test_noobaa uninstall --cleanup 
+        check_if_cleanup
+    else
+        echo_time "Running uninstall without --cleanup”"
+        test_noobaa uninstall 
+    fi  
+}
+
+function check_if_cleanup {  
+    crd_array_after_Cleanup=($(kubectl get crd | awk '{print $1}' | grep -v "NAME"))   
+    for should_be_empty_crd_array in ${crd_array[@]}
```

Comment:
should_be_empty_crd_array is not an array.
```suggestion
    for crd_before_clean in ${crd_array[@]}
```